src/auth/auth-token-integration.ts

```diff
@@ -7,6 +8,9 @@ export class BBAuthTokenIntegration {
     permissionScope?: string,
     leId?: string
   ) {
+    if (!this.getLocationHostname().endsWith('blackbaud.com')) {
```

Comment:
I don't believe `endsWith` is available in all the browsers we support, nor do we have the polyfill for it.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith#Browser_compatibility